packages/theme-a11y/theme-a11y.js

```diff
@@ -199,3 +199,75 @@ export function removeTrapFocus() {
   document.removeEventListener('focusout', trapFocusHandlers.focusout);
   document.removeEventListener('keydown', trapFocusHandlers.keydown);
 }
+
+/**
+ * Add a preventive message to external links and links that open to a new window.
+ * @param {object} elements - Specific elements to be targeted
```

Comment:
Right now `elements` has to be an array because you have a `forEach` down below.  I would rather this be a string selector and we do the `querySelectorAll` on our end.  As opposed to checking if they gave us an array.